test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -451,7 +451,8 @@ def _run_dependency_check_test(image, ec2_connection):
         "autogluon": {
             "0.3": ["cpu", "gpu"],
```

Comment:
Do you still want to keep versions that are no longer supported?

Reply:
removed